rllib/evaluation/rollout_worker.py

```diff
@@ -1499,56 +1500,55 @@ def get_filters(self, flush_after: bool = False) -> Dict:
         return return_filters
 
     @DeveloperAPI
-    def get_state(self) -> bytes:
+    def get_state(self) -> dict:
         """Serializes this RolloutWorker's current state and returns it.
 
         Returns:
             The current state of this RolloutWorker as a serialized, pickled
             byte sequence.
         """
         filters = self.get_filters(flush_after=True)
-        state = {}
-        policy_specs = {}
-        connector_enabled = self.policy_config.get("enable_connectors", False)
+        policy_states = {}
         for pid in self.policy_map:
-            state[pid] = self.policy_map[pid].get_state()
-            policy_spec = self.policy_map.policy_specs[pid]
-            # If connectors are enabled, try serializing the policy spec
-            # instead of picking the spec object.
-            policy_specs[pid] = (
-                policy_spec.serialize() if connector_enabled else policy_spec
-            )
-        return pickle.dumps(
-            {
-                "filters": filters,
-                "state": state,
-                "policy_specs": policy_specs,
-                "policy_config": self.policy_config,
-            }
-        )
+            policy_states[pid] = self.policy_map[pid].get_state()
+        return {
+            "filters": filters,
+            # TODO: Rename this into "policy_states" for clarity.
+            "state": policy_states,
```

Comment:
New: Policy's config and spec are now part if its own state and therefore need no longer be stored separately from the Policy states in the Worker's state dict!
This allows us to restore an individual policy from a policy checkpoint w/o requiring the original config, action space, etc..